src/builtins.c

```diff
@@ -966,10 +967,9 @@ JL_CALLABLE(jl_f_invoke)
     JL_GC_PUSH1(&argtypes);
     if (!jl_is_tuple_type(jl_unwrap_unionall(args[1])))
         jl_type_error("invoke", (jl_value_t*)jl_anytuple_type_type, args[1]);
-    if (!jl_tuple_isa(&args[2], nargs-2, (jl_datatype_t*)argtypes))
+    if (!jl_tuple_isa(args[2], &args[3], nargs - 2, (jl_datatype_t*)argtypes))
```

Comment:
It doesn't seem like this should change; the first argument isn't the called function, and `jl_tuple_isa` is internal.